### ISSUE TITLE
how can i improve the accuracy for this model and could anyone help me to increase the loss percentage 

### ISSUE THREAD

**Issue by devendraswamy** [jwyang/faster-rcnn.pytorch#732]
how can i improve the accuracy for this model and could anyone help me to increase the loss percentage. 

Thanking you !

**Comment by devendraswamy** [comment]
[session 1][epoch 25][iter  100/ 107] loss: 0.3858, lr: 1.00e-07
			fg/bg=(4/1020), time cost: 2159.435124
			rpn_cls: 0.0213, rpn_box: 0.1552, rcnn_cls: 0.0151, rcnn_box 0.0000
save model: models/vgg16/pascal_voc/faster_rcnn_1_25_106.pth

please tell any suggestion for decrease the accuracy

**Comment by kangbyounggwan** [comment]
It's too late. Reduce your learning rate!! 
lr=0.002 epoch 3 times
I got a loss of 0.55 !!
